daemon/mgr/container.go

```diff
@@ -99,40 +99,38 @@ func NewContainerManager(ctx context.Context, store *meta.Store, cli *ctrd.Clien
 // Restore containers from meta store to memory and recover those container.
 func (mgr *ContainerManager) Restore(ctx context.Context) error {
 	fn := func(obj meta.Object) error {
-		c, ok := obj.(*types.ContainerInfo)
+		c, ok := obj.(*Container)
```

Comment:
the object be stored in metastore is "*ContainerMeta", not "*Container". so

c, ok := obj.(*mgr.ContainerMeta)